### ISSUE TITLE
Test producto nombre string vacio

### ISSUE THREAD

**Issue by mariomerlano** [frlp-utn-is-2018/orderingg#67]
Hacer un test de unidad para verificar que no esté permitido la creación de un producto con el nombre como string vacío

    def test_order_product_name_vacio(self):
        
        #Creo un producto
        producto = {
            'id':1,
            'name': '',
            'price': 50
        }
       
        self.client.post('/product', data=json.dumps(producto), content_type='application/json')

        #Creo una orden
        order = {
                        "id": 1 
                }
        
        order = Order()
        #Guardo la orden en la db directo ya que no está en endpoint en la api
        db.session.add(order)
        db.session.commit()

        orderProduct =  {"quantity":1,"product":{"id":1}}

        resp = self.client.post('/order/1/product', data=json.dumps(orderProduct), content_type='application/json')
        self.assert200(resp, "Falló el POST")

Respuesta:
E   AssertionError: 201 != 200 : Falló el POST

Mi duda es la siguiente.. que es lo que tengo que comprobar? Como hago para no permitir que se cree un producto con el nombre vacio? tendria que poner algun if verificando el nombre?

**Comment by RodrigoJacznik** [comment]
El Server retorna un status code `201 created` lo cual es correcto. Lo que está mal es tu assert

**Comment by mariomerlano** [comment]
La cuestion es que no se tendria que crear porque esta el nombre vacio, como impido eso?

**Comment by RodrigoJacznik** [comment]
Claro, es un bug. Podrías testear que no retorne ese status code y que no quede en la base de datos.
El test debería fallar
